package.json

```diff
@@ -99,9 +99,7 @@
     "date-fns-tz": "^2.0.0",
     "dom-serializer": "^0.2.2",
     "domhandler": "^4.3.0",
-    "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#398bf7c6a6d37f229a41d92bd7a4324c0fd32849",
-    "expo": "^49.0.0",
-    "expo-image": "1.8.1",
```

Comment:
why are we removing the expo packages in this PR?

Reply:
Oh I didn't notice that, probably merge conflicts. We should only be modifying `expensify-common`

Reply:
Ok, can you please fix that then @byte1012

Reply:
my mistake of merge, fixed now @puneetlath @s77rt